fastFM/ffm2.pyx

```diff
@@ -48,3 +48,51 @@ def ffm_predict(double w_0, double[:] w,
     del d
 
     return y
+
+def ffm_als_fit(fm, X, double[:] y):
```

Comment:
I would suggest to add explicit parameter for the fm attributes used in `ffm_als_fit`: `fm.rank` and `fm.warm_start` instead of `fm`. This way we have less coupling (I only realized this when checking why `from fastFM import als` is needed in the test.

Reply:
I just saw that you followed my `ffm.ffm_als_fit(fm, X, y)` signature. I should better review my own code :-) .